core/sawtooth/manage/subproc.py

```diff
@@ -59,13 +51,10 @@ def _construct_start_command(self, node_args):
             config_dir = os.path.join(node_args.currency_home, 'etc')
```

Comment:
Same as above for daemon:
"config_dir" doesn't seem to even be used elsewhere in the code now. Should the creation of this directory, and references to it, just be removed altogether?